Minimum package coverage map

VitalyNikonorov opened the pull request:
Added possibility to configure custom test coverage value for whole specific package.
E.g. added new package 'com/company/newmodule' and you need to add specific test coverage just for this package.

Malinskiy commented:
Hey @VitalyNikonorov! Thanks for your effort!
Could you please add tests to your code to make sure it's behaving as expected? Thanks!

VitalyNikonorov commented:
> Hey @VitalyNikonorov! Thanks for your effort!
> Could you please add tests to your code to make sure it's behaving as expected? Thanks!

@Malinskiy 
Fixed semantic ruby error and added tests for new functionality
